newton/examples/robot/example_robot_allegro_hand.py

```
@@ -123,30 +123,34 @@ def __init__(self, viewer, num_envs=4):
+                self.model,
+                solver="newton",
+                integrator="euler",
+                njmax=200,
+                ncon_per_env=150,
+                impratio=10.0,
+                cone="elliptic",
+                iterations=100,
+                ls_iterations=50,
+                use_mujoco_cpu=False,
+            )
+        import sys; sys.exit()
```

Ruff (0.12.2)

Multiple statements on one line (semicolon)

(E702)